extensions/machine-learning/src/views/models/azureModelsComponent.ts

```diff
@@ -49,9 +52,32 @@ export class AzureModelsComponent extends ModelViewBase implements IDataComponen
 			this._downloadedFile = undefined;
 		});
 
+		this._emptyModelsComponent = new DataInfoComponent(this._apiWrapper, this);
+		this._emptyModelsComponent.width = 300;
+		this._emptyModelsComponent.height = 250;
+		this._emptyModelsComponent.iconSettings = {
+			css: { 'padding-top': '20px' },
+			width: 100,
```

Comment:
The empty state icon (documents and magnifying glass) is 128x128.

Reply:
should I change the height to 128?